climada/util/config.py

```diff
@@ -321,3 +312,4 @@ def _fetch_conf(directories, config_name):
     Path(Path.home(), '.config'),  # ~/.config directory
     Path.cwd(),  # current working directory
 ], CONFIG_NAME))
+LOGGER.setLevel(getattr(logging, CONFIG.log_level.str()))
```

Comment:
`config.py` is the file where both `LOGGER` and `CONFIG` are initialized. It doesn't make sense to me to make the final step of setting to log-level of `LOGGER` according to `CONFIG` outside of this file. It should happen right here, once the `CONFIG` is loaded.